yogstation/code/modules/mob/living/simple_animal/friendly/goats.dm

```diff
@@ -0,0 +1,347 @@
+/mob/living/simple_animal/hostile/retaliate/goat/clown
+	name = "Gary the Goat"
+	desc = "Rather kick your butt then tell jokes."
```

Comment:
than.

Well, I suppose then could also be used, but I'm about 99% sure you meant to write than.